assembl/static2/js/app/components/common/visibilityComponent.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class VisibilityComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVisible: props.isVisible
+    };
+  }
+  render() {
+    const { className } = this.props;
+    const { isVisible } = this.state;
+    const myClassNames = isVisible === true ? className : `${className} hidden`;
```

Comment:
now we can use the `classnames` module for such stuff : https://www.npmjs.com/package/classnames (we added the dependency in the richtexteditor branch)